### PR TITLE
Add display flag for pillar config

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

**Comment by riecke** [comment]
Did you run the tests?  "python manage.py \*test\*.py"

**Comment by elsiel23** [comment]
I ran them via Visual Studio pytest discovery

File: src/planscape/config/conditions_config.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def check_region(region) -> bool:
 
         def check_pillar(pillar) -> bool:
             return (isinstance(pillar, dict) and
-                    pillar.keys() <= {'pillar_name', 'elements', 'operation', 'filepath', 'display_name'} and
+                    pillar.keys() <= {'pillar_name', 'elements', 'operation', 'filepath', 'display_name','display'} and
```
Comment (thequeenofspades) [inline src/planscape/config/conditions_config.py:50]: nit: should be a space before 'display'

File: src/planscape/config/conditions.json [hunk 2]
```diff
@@ -191,6 +191,7 @@
                     "pillar_name": "air_quality",
                     "filepath": "data/sierra_nevada/air_quality",
                     "display_name": "Air Quality",
+                    "display": "False",
```
Comment (thequeenofspades) [inline src/planscape/config/conditions.json:194]: can we represent this as a bool instead of a string?

**Review by thequeenofspades — APPROVED** [review]

File: src/planscape/config/conditions_config.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/config/conditions_config.py:50]: Done

File: src/planscape/config/conditions.json (hunk 2 above)
Comment (elsiel23) [reply src/planscape/config/conditions.json]: Done!

**Pull request merged**